src/App.tsx

```diff
@@ -15,8 +12,11 @@ import '@patternfly/patternfly/utilities/Text/text.css';
 import { Suspense } from 'react';
 import { BrowserRouter as Router } from 'react-router-dom';
 import 'reactflow/dist/style.css';
+import './App.css';
+import { useIntegrationJsonStore } from './store';
+import { bindUndoRedo } from './utils';
 
-const { undo, redo } = useTemporalIntegrationJsonStore.getState();
+const { undo, redo } = useIntegrationJsonStore.temporal.getState();
```

Comment:
According to the API, this is the same as before, except that in this case, we're accessing the underlying temporal store.

I'm not sure how to test the behavior to be completely sure that it's ok, any idea is welcomed :dancers: